app/controllers/api/validations_controller.rb

```diff
@@ -1,9 +1,16 @@
 module Api
   class ValidationsController < ApiController
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Disabling metrics inline seems… not good… to me. Did you intend to commit this?

(The method length cop disabled here is usually a great indicator for 'you could probably describe your intent better by extracting methods'.)